rllib/algorithms/cql/cql_learner.py

```diff
@@ -0,0 +1,25 @@
+from ray.air.constants import TRAINING_ITERATION
+from ray.rllib.algorithms.sac.sac_learner import SACLearner
+from ray.rllib.core.learner.learner import Learner
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.metrics import ALL_MODULES
+
+
+class CQLLearner(SACLearner):
+    @override(Learner)
+    def build(self) -> None:
+        # We need to call the `super()`'s `build` method here to have the variables
+        # for `alpha`` and the target entropy defined.
+        super().build()
+
+        # Set up the gradient buffer to store gradients to apply
+        # them later in `self.apply_gradients`.
+        self.grads = {}
+
+        # Log the training iteration to switch from behavior cloning to improving
+        # the policy.
+        # TODO (simon, sven): Add upstream information pieces into this timesteps
+        # call arg to Learner.update_...().
+        self.metrics.log_value(
```

Comment:
Add: reduce="sum" here to make sure iterations are not EMA'd :) (the default).

Reply:
Actually, you don't need this here anymore. B/c you are logging it below, this entry will be created automatically.